src/Serilog.Sinks.Email/LoggerConfigurationEmailExtensions.cs

```diff
@@ -42,7 +42,7 @@ public static class LoggerConfigurationEmailExtensions
         /// <param name="batchPostingLimit">The maximum number of events to post in a single batch.</param>
         /// <param name="period">The time to wait between checking for event batches.</param>
         /// <param name="formatProvider">Supplies culture-specific formatting information, or null.</param>
-        /// <param name="mailSubject">The subject used in error mails</param>
+        /// <param name="mailSubject">The subject used in error mails, can be a plain string or a template such as {Timestamp} [{Level}] occurred...</param>
```

Comment:
While we're here, perhaps we can fix the comment (they're not always errors) and use:

```
The email subject, can be a plain string or a template such as "{Timestamp} [{Level}] occurred".
```

(Also, single full-stop `.`)